hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -241,13 +241,18 @@ static const struct usage_map xpadneo_usage_maps[] = {
 	USAGE_IGN(0x9001E), /* copy of RS */
 	USAGE_IGN(0xC0082), /* copy of Select button */
 
-	/* XBE2: Disable unused "TRIGGER_HAPPY" buttons */
-	USAGE_IGN(0x90012),
+	/* XBE2: Disable extra features until proper support is implemented */
+	USAGE_IGN(0xC0081), /* Four paddles */
+	USAGE_IGN(0xC0085), /* Profile switcher */
```

Comment:
The profile switcher exposes itself as a HID event? Than implementation may become more tricky as we may need to hook-up some state change to this. Nevertheless, it looks kinda redundant to me, as we also get the profile ID in the raw report data.